ui/page/components/account_selector.go

```diff
@@ -33,14 +36,16 @@ type AccountSelector struct {
 	selectedWalletName string
 	totalBalance       string
 	changed            bool
+	subbedTxNotif      bool
```

Comment:
by `subbedTxNotif` you mean? maybe an inline comment can help out here.
also i think `subbedTxNotification` is better.